.travis.yml

```diff
@@ -9,10 +11,38 @@ os:
    - osx
 
 env:
-   - DEBUG=YES MPI=YES TMPDIR=/tmp
-   - DEBUG=NO MPI=YES TMPDIR=/tmp
-   - DEBUG=YES MPI=NO
-   - DEBUG=NO MPI=NO
+   global:
```

Comment:
Enabled the full 'test' target on our debug builds instead of 'check', to allow code coverage to get results of all the parallel and serial tests.